plugins/polio/js/src/domains/Calendar/campaignCalendar/Filters.tsx

```diff
@@ -1,29 +1,44 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React, { useState, useEffect, useCallback, useMemo } from 'react';
+import React, {
+    useState,
+    useEffect,
+    useCallback,
+    useMemo,
+    FunctionComponent,
+} from 'react';
 import { FormattedMessage } from 'react-intl';
 import { useDispatch } from 'react-redux';
-import PropTypes from 'prop-types';
 import { replace } from 'react-router-redux';
 
 import { Grid, Button, Box, useMediaQuery, useTheme } from '@mui/material';
 import FiltersIcon from '@mui/icons-material/FilterList';
 import { withRouter } from 'react-router';
-import { useSafeIntl } from 'bluesquare-components';
-import InputComponent from 'Iaso/components/forms/InputComponent.tsx';
-import DatesRange from 'Iaso/components/filters/DatesRange';
-import { useGetGroupDropdown } from 'Iaso/domains/orgUnits/hooks/requests/useGetGroups.ts';
+import { IntlMessage, useSafeIntl } from 'bluesquare-components';
+import { Router } from '../../../../../../../hat/assets/js/apps/Iaso/types/general';
+import InputComponent from '../../../../../../../hat/assets/js/apps/Iaso/components/forms/InputComponent';
+import DatesRange from '../../../../../../../hat/assets/js/apps/Iaso/components/filters/DatesRange';
+import { useGetGroupDropdown } from '../../../../../../../hat/assets/js/apps/Iaso/domains/orgUnits/hooks/requests/useGetGroups';
 import MESSAGES from '../../../constants/messages';
 import { useGetCountries } from '../../../hooks/useGetCountries';
-import { useGetGroupedCampaigns } from '../../GroupedCampaigns/hooks/useGetGroupedCampaigns.ts';
+import { useGetGroupedCampaigns } from '../../GroupedCampaigns/hooks/useGetGroupedCampaigns';
 
-import { genUrl } from '../../../../../../../hat/assets/js/apps/Iaso/routing/routing.ts';
+import { genUrl } from '../../../../../../../hat/assets/js/apps/Iaso/routing/routing';
 import {
     dateApiToDateRangePicker,
     dateRangePickerToDateApi,
-} from '../../../../../../../hat/assets/js/apps/Iaso/utils/dates.ts';
-import { appId } from '../../../constants/app.ts';
+} from '../../../../../../../hat/assets/js/apps/Iaso/utils/dates';
+import { appId } from '../../../constants/app';
+import { CalendarParams } from './types';
 
-const campaignTypeOptions = (formatMessage, showTest = false) => {
+type Props = {
+    router: Router & { params: CalendarParams };
+    disableDates: boolean;
+    disableOnlyDeleted: boolean;
+    isCalendar: boolean;
+    showTest: boolean;
+};
+
+const campaignTypeOptions = (formatMessage: IntlMessage, showTest = false) => {
```

Comment:
```suggestion
const campaignTypeOptions = (formatMessage: IntlFormatMessage, showTest = false) => {
```

Wrong type. There's a typo in your type (hu hu)